cmd/up/up.go

```diff
@@ -338,7 +339,8 @@ func Up() *cobra.Command {
 				}()
 			}
 
-			if err := setBuildEnvVars(ctx, oktetoManifest); err != nil {
+			// build images and set env vars for the services at the manifest
+			if err := buildAllServices(ctx, oktetoManifest, up.builder); err != nil {
```

Comment:
if I call `buildAllService` I expect to build all services but inside the func there is logic to check which services needs to be build. Can we rename the func -> `buildServices` or `buildNeededServices` ?

Reply:
agree on the "all" i was thinking on all as iteration on all the services of the manifest, but the build can be done/not if is already built. I've renamed and modified the comment